<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VtImageSetDims</class>
 <widget class="QDialog" name="VtImageSetDims">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Dimensions</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>415</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="widthComboBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>55</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="widthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QComboBox" name="heightComboBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>30</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="depthComboBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>80</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="depthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Depth</string>
   </property>
  </widget>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthSpinBoxStart">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>57</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthSpinBoxEnd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>57</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="endLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="strideLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stride</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthSpinBoxStride">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>57</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="widthMaxLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>55</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightSpinBoxStart">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>32</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightSpinBoxEnd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>32</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightSpinBoxStride">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>32</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="heightMaxLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="depthSpinBoxStart">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>82</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="depthSpinBoxEnd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>82</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="depthSpinBoxStride">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>82</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="depthMaxLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="rangeLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VtImageSetDims</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VtImageSetDims</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
